shark/shark_importer.py

```diff
@@ -117,26 +110,52 @@ def load_tflite_model(self):
                 )
                 tflite_model_list = csv.reader(open(tflite_model_list_path))
                 for row in tflite_model_list:
-                    if str(row[0]) == self.model_name:
+                    if str(row[0]) == str(self.model_name):
                         self.model_path = row[1]
+                        print("tflite_model_name", str(row[0]))
+                        print("tflite_model_link", self.model_path)
             if self.model_path is None:
                 print("Error, No model path find in tflite_model_list.csv")
                 return False
-            urllib.request.urlretrieve(self.model_path, self.tflite_saving_file)
-        if os.path.exists(self.tflite_tosa_file):
-            print("Exists", self.tflite_tosa_file)
+            urllib.request.urlretrieve(self.model_path, self.raw_model_file)
+        if os.path.exists(self.mlir_file):
+            print("Exists MLIR model ", self.mlir_file)
         else:
             print(
                 "No tflite tosa.mlir, please use python generate_sharktank.py to download tosa model"
             )
+            print("Convert tflite to tosa.mlir")
+            ireec_tflite.compile_file(
+                self.raw_model_file,
+                input_type="tosa",
+                save_temp_iree_input=self.mlir_file,
+                target_backends=[IREE_TARGET_MAP["cpu"]],
+                import_only=False,
+            )
+        with open(self.mlir_file) as f:
+            self.mlir_model = f.read()
         return True
 
+    def setup_interpreter(self):
```

Comment:
All the tf-lite utils should go into tf_lite_utils.py.

Reply:
Done